tests/integration/self_update_test.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+
+include './src/lib/kwio.sh'
+include './src/lib/kwlib.sh'
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+
+declare -a KW_BRANCHES=('master' 'unstable')
+declare -g HOST_KW_CLONED_PATH
+
+function oneTimeSetUp()
+{
+  local kw_repo_url='https://github.com/kworkflow/kworkflow'
+
+  HOST_KW_CLONED_PATH="$(mktemp --directory)/kworkflow"
```

Comment:
I think this is a nitpick, but maybe we want to create another readonly global variable to store the path returned by `mktemp --directory`, so we can remove this temporary dir in `oneTimeTearDown` or below, in case of fail. At the moment, we are only removing the `kworkflow` repo inside the temp dir (again, a nitpick, as this temp dir wont occupy practically any space...)

Reply:
Nice catch, I added this change :)